### PR TITLE
fix wording for apps mgmt buttons

### PR DESCRIPTION
**Pull request by jancborchardt**
- Ridiculous »Please wait....« → »Enabling app …«
- Capitalization fix

Please check @nextcloud/designers

### CODE REVIEW

File: settings/js/apps.js [hunk 1]
```diff
@@ -271,7 +271,7 @@ OC.Settings.Apps = OC.Settings.Apps || {
 		OC.Settings.Apps.hideErrorMessage(appId);
 		groups = groups || [];
 		var appItem = $('div#app-'+appId+'');
-		element.val(t('settings','Please wait....'));
+		element.val(t('settings','Enabling app …'));
```
Comment (skjnldsv) [inline settings/js/apps.js]: Space before the dots? Typo?

**Comment by jancborchardt** [comment]
@nickvergessen I looked at the failures but can’t tell why it fails. Can you help me?

File: settings/js/apps.js (hunk 1 above)
Comment (skjnldsv) [inline settings/js/apps.js]: Seems weird to me, in French we use it the other way, if it replace a word, we add a space, if it indicate a continuing thing, we don't add a space. :)

**Comment by nickvergessen** [comment]
Ah it's the strange `TypeError: undefined is not a function (evaluating 'this.data.state()')` again, so ignore. I thought it was your string changes

**Comment by nickvergessen** [comment]
Please backport to stable11?

File: settings/js/apps.js (hunk 1 above)
Comment (jancborchardt) [inline settings/js/apps.js]: In this case it’s neither though – it indicates progress. Anyway that’s why it’s translatable and the French language team can translate it properly. ;) You also put a space before exclamation marks which typographically looks horrible !

**Pull request merged**